source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -548,19 +548,19 @@ Once your app is submitted, an Asana developer will configure and enable your ap
 |» `x96`                      | String (url) | 96x96 icon asset |
 |» `x192`                     | String (url) | 192x192 icon asset |
 | `capabilities`              | Object       | A list of capabilities supported by the app and their configuration. |
-|» `resource_widget`          | String (url) | The container for resource widget functionality  |
-|»» `widgetMetadataUrl`       | String (url) | A URL that Asana uses to make requests for the data needed to load an app widget which displays information about a 3rd party resource. |
-|»» `matchUrlPattern`         | String (url) | A regex which allows Asana to compute whether a UrlAttachment is supported by an activated app on the project in order to render an app widget. |
+|» `resource_widget`          | String (url) | The container for resource Widget functionality  |
```

Comment:
Unrelated, but this is an Object, not a "String (url)"

Reply:
Thanks for catching that!